theano/gof/opt.py

```diff
@@ -201,7 +195,12 @@ def warn(exc, self, optimizer):
 
     def __init__(self, *opts, **kw):
         """
-        WRITEME
+        Parameters
+        ----------
+        opts : List
```

Comment:
Actually no, this is not a list argument.  This it the list of all arguments.  Just document it as `*opts` with no type.